Server/mods/deathmatch/logic/luadefs/CLuaDatabaseDefs.cpp

```diff
@@ -754,16 +754,16 @@ int CLuaDatabaseDefs::ExecuteSQLSelect(lua_State* luaVM)
         CPerfStatSqliteTiming::GetSingleton()->SetCurrentResource(luaVM);
         if (CStaticFunctionDefinitions::ExecuteSQLSelect(strTable, strColumns, strWhere, uiLimit, &Result))
         {
-            lua_newtable(luaVM);
-            int i = 0;
-            for (CRegistryResultIterator iter = Result->begin(); iter != Result->end(); ++iter, ++i)
+            lua_createtable(luaVM, Result->size(), 0);
+            lua_Number i = 1;
```

Comment:
Same feedback to share in executeSqlQuery

Reply:
This is the rows table.
It'll be filled by the exact amount of rows even if all columns consist of nulls.